libs/ui-lib/src/lib/styles/styles.scss

```diff
@@ -11,6 +11,13 @@
   border: 0;
 }
 
+html,
+body,
+#root {
+  width: 100%;
+  height: 100%;
+}
+
```

Comment:
I don't like this, have you considered how it will affect other styling? 
I see you're doing this to keep the button at the bottom I know there was a discussion on if we're keeping it sticky or part of the scrolling section. If it needs to stick to the bottom, consider position abolsute.

Reply:
This isn't to have the button stuck at the bottom. This is to make the event info component scroll correctly doable. For the event info to scroll but not the event page itself, the event page needs to take up the whole page, which wasn't possible since `html`, `body` and `#root` had `fit-content` as their height.

I really doubt this will negatively affect any styling imo, since all it does is make sure that containers take up the entire page, which I thought it was doing already. Also I tested the existing page and no stylng broke as far as I know. 

This warrants a longer conversation, thought, and obviously I might be wrong on what I said earlier. Open to alternatives since you have more experience than me on this.

Reply:
found an alternative solution. The height of the page is 100vh - height of navbar - height of header

Reply:
Nice! That's what I was getting at with `vh` very useful!